contents/architecture/RESTArchitecturalStyle.md

```diff
@@ -0,0 +1,226 @@
+# REST Architectural Style
+
+Author: [Wen Xin](https://github.com/wenmogu)
+
+This book chapter introduces the REST architectural style, which is the guideline of development for the Web we use today. The following content is a summary of selected parts from the famous [Roy Fielding's paper about REST](https://www.ics.uci.edu/~fielding/pubs/dissertation/fielding_dissertation.pdf) with my own understanding, which I hope can serve as a relatively brief, easy-to-understand and at the same time detailed introduction to REST.
+
+**Requirement of Web development --- Background of REST architectural style**
+
+The forefather of Web Sir Tim Berners-Lee envisioned the Web to be &quot;a shared information space through which people and machines could communicate.&quot; Fielding&#39;s understanding of &quot;this shared information space&quot; is that the Web is a place where everyone&#39;s information is structured and stored in a way such that their own information &quot;could be usable to themselves and others&quot;, and they can &quot;reference and structure&quot; others&#39; information such that they do not need to keep a local copy of it.
```

Comment:
Since there are several inline quotes present within this single paragraph, why not extract everything into a blockquote, followed by your analysis afterwards? This can allow for cleaner presentation, as well as provide a clearer structure for the content. It may also help with the issue where you seem to be chaining too many quotes together, resulting in long sentences that are somewhat difficult to digest (e.g. the second sentence which spans three lines).